src/components/organisms/Root.tsx

```diff
@@ -0,0 +1,9 @@
+import { useSelector } from 'react-redux';
+import { selectActiveWorkspace } from '../../state/workspaces/selectors';
+import { WorkspaceSelector } from './WorkspaceSelector';
+import { Workspace } from './Workspace';
+
+export function Root() {
```

Comment:
This is separate from App because you need to be within the Provider context to use Selectors and whatnot. I'm open to renaming it.